packages/backend/src/plugins/k8s-apply.ts

```diff
@@ -125,7 +125,10 @@ export const createKubernetesApply = (config: Config) => {
             },
           ],
         };
-        if (!confFile.clusters[0].cluster['insecure-skip-tls-verify']) {
+        if(confFile.clusters[0].cluster['insecure-skip-tls-verify']) {
+          // Remove insecure-skip-tls-verify from k8s configfile
+          delete confFile.clusters[0].cluster['insecure-skip-tls-verify']
```

Comment:
Confused here. Why are we removing this field and how does it help with the issue you have?

Reply:
We are adding caData (from /var/run/secrets/kubernetes.io/serviceaccount/ca.crt) to confFile without encryption, which is causing the issue. This CA is needed only when we set 'insecure-skip-tls-verify' as **false**. So we are removing that field from confFile if we specify **skipTLSVerify** as **true**.

https://github.com/cnoe-io/backstage-app/blob/main/packages/backend/src/plugins/k8s-apply.ts#L109-L110

Reply:
I have corrected @nabuskey

Reply:
Could you please check PR now @nabuskey